core/tests/webdriverio_utils/TopicEditorPage.js

```diff
@@ -264,12 +264,10 @@ var TopicEditorPage = function() {
 
   this.addConceptCardToSubtopicExplanation = async function(skillName) {
     await action.click('RTE input', subtopicPageContentButton);
-    // The cke buttons classes are dynamically alloted and hence we cannot
-    // a class name to select a particular button. Also using chain selectors
-    // is returning a non interactable element hence we are explicitly using
-    // the dynamically alloted id of the button to select the button.
+    // The cke buttons classes are dynamically alloted so we cannot
+    // add class name starting with 'e2e' to them.
     // eslint-disable-next-line oppia/e2e-practices
-    var conceptCardButton = $('#cke_124');
+    var conceptCardButton = $('.cke_button*=Insert Concept Card Link');
```

Comment:
The Id's are alloted dynamically to the cke buttons, after the changes in the PR, the id of concept was updated to cke_118 instead of cke_124 causing the tests to [fail](https://github.com/oppia/oppia/runs/7892281673?check_suite_focus=true). So instead of any such id I am now using a user defined class, which will not cause any such failure in future.

Reply:
Do you know why the tests never failed in protractor due to this reason?

Reply:
In protractor we were not using ID, instead we were using chain selectors which was not working with the webdriverio (we discussed this in meet), so I used the specific ID present.